cesm/driver/ensemble_driver.F90

```diff
@@ -9,7 +9,7 @@ module Ensemble_driver
 
   use shr_kind_mod  , only : cl=>shr_kind_cl, cs=>shr_kind_cs
   use shr_log_mod   , only : shr_log_setLogUnit
-  use esm_utils_mod , only : mastertask, logunit, chkerr
+  use esm_utils_mod , only : maintask, logunit, chkerr
```

Comment:
It seems that you converted all master->main. Did you do it intentionally or just run a script to replace them. Just wonder. It is okay for me if you replay want to change master to main in everywhere

Reply:
Yes, I just used a script, but my intent was to replace master with main in all cases.  Except possibly in OpenMP directives which as far as I can tell still use master slave language.

Reply:
Okay. That sounds fine. Thanks.